packages/utils/src/snark-artifacts/snark-artifacts.browser.ts

```diff
@@ -1,31 +1,21 @@
-import { GetSnarkArtifactUrls } from "./config"
-import { Proof, SnarkArtifacts, Version } from "../types"
+import { BigNumber, SnarkArtifacts, Version } from "../types"
 
-function MaybeGetSnarkArtifacts(proof: Proof.EDDSA, version?: Version): () => Promise<SnarkArtifacts>
-function MaybeGetSnarkArtifacts(
-    proof: Proof.POSEIDON,
-    version?: Version
-): (numberOfInputs: number) => Promise<SnarkArtifacts>
-function MaybeGetSnarkArtifacts(
-    proof: Proof.SEMAPHORE,
-    version?: Version
-): (treeDepth: number) => Promise<SnarkArtifacts>
-function MaybeGetSnarkArtifacts(proof: Proof, version?: Version) {
-    switch (proof) {
-        case Proof.POSEIDON:
-            return async (numberOfInputs: number) => GetSnarkArtifactUrls({ proof, numberOfInputs, version })
+export default async function maybeGetSnarkArtifacts(
+    projectName: string,
+    options: {
+        parameters?: (BigNumber | number)[]
+        version?: Version
+        cdnUrl?: string
```

Comment:
I'd like to hear your thoughts about https://github.com/privacy-scaling-explorations/zk-kit/pull/273

Reply:
Yess, I think that's the right way to do it! Sounds like a solid mechanism 👍🏽